src/manifest/configPlugins.ts

```diff
@@ -1,29 +1,30 @@
 import {
   resolveConfigPluginFunction,
-  resolvePluginForModule,
+  resolveConfigPluginFunctionWithInfo,
 } from '@expo/config-plugins/build/utils/plugin-resolver';
+import fs from 'fs';
```

Comment:
This works, but we might want to take a look at [`vscode.fs`](https://code.visualstudio.com/api/references/vscode-api#FileSystem). It should remove possible issues when using remote machines to develop with in vscode

Reply:
ok